CryptoPkg/Private/Protocol/Crypto.h

```diff
@@ -5707,6 +5835,7 @@ struct _EDKII_CRYPTO_PROTOCOL {
   EDKII_CRYPTO_EC_POINT_SET_AFFINE_COORDINATES        EcPointSetAffineCoordinates;
   EDKII_CRYPTO_EC_POINT_ADD                           EcPointAdd;
   EDKII_CRYPTO_EC_POINT_MUL                           EcPointMul;
+  EDKII_CRYPTO_EC_POINT_MUL2                          EcPointMul2;

```

Comment:
1. Need to append new items at end with (Continued).
2. Need to update EDKII_CRYPTO_VERSION after protocol changes.